src/index.html

```diff
@@ -7,8 +7,55 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="container">
+      <a href="#home">
+        <img
+          class="logo__img"
+          src="moyo_1.png"
+          alt="logo">
```

Comment:
```suggestion
        <img
          class="logo__img"
          src="moyo_1.png"
          alt="logo"
        >
```
pay attention to indentation